packages/consent/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import { loadPrivacyManagerModal } from './sourcepoint';
```

Comment:
We'd need the consent library to be app and vendor-list agnostic. And that makes type-safety of the APIs more complex to achieve especially with `useGrant` hooks. Also, we have those setups/configurations that need to happen for consent to work

All of these are mostly speculation but, I think we should take more time for the consent module, maybe have a separate ticket and do more design discussion before we move to it. I think we can keep the in-place consent module pending when this is ready,

Reply:
Let's create tickets for fully migrating consent to @goodhood/consent!